ProjectSettings/ProjectSettings.asset

```diff
@@ -3,7 +3,7 @@
 --- !u!129 &1
 PlayerSettings:
   m_ObjectHideFlags: 0
-  serializedVersion: 20
```

Comment:
could you exclude the projectsettings etc. from the PR please?

Reply:
Oops. not sure how those snuck in